lib/app.rb

```diff
@@ -52,11 +52,11 @@ def run_commands
 
       commands = @display.get_user_commands
       system 'clear'
-      @simulator.evaluate(commands)
-      @simulator.board.print_board
-      puts 'Feel free to enter another set of commands'
+      @simulator.evaluate(commands) { |outcome| @display.print_outcome(@simulator.robot, @simulator.board, outcome) }
```

Comment:
again, is `.print_outcome` a generic enough responsibility of `Display` ? so far I am thinking
1. ~~print_screen~~ -> redraw
2. ~~print_error~~ -> notify_error
3. maybe this is print_prompt? or ~~print_action~~ -> notify

_I have crossed out the print's as maybe they are not always printed, say in a web app_